src/window/settings.rs

```diff
@@ -16,6 +17,7 @@ pub struct WindowSettings {
     pub touch_deadzone: f32,
     pub touch_drag_timeout: f32,
     pub background_color: String,
+    pub background_image: String,
```

Comment:
Should this be in `src/renderer/mod.rs` in `RendererSettings` instead? (alongside with `background_color`)

Reply:
I think it makes more sesnse to keep it in window settings as it modifies the window property rather than how the window is being rendered